### PR TITLE
Adding today's homework Assignment

### PR DESCRIPTION
**Pull request by PoonamDDec19**

### CODE REVIEW

File: src/poonamDPracticeTest/StringNumLetters.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		String SmallLetters = "";
+		String CapitalLetters = "";
+		String SpclChar = "";
+		int OddSum = 0 ;
+		int EvenSum = 0;
+		for(int i=0;i<str.length();i++)
+		{ 
+			char ch = str.charAt(i);
+			boolean isNum = true;
+			boolean isLetter = true;
```
Comment (mkanani13) [inline src/poonamDPracticeTest/StringNumLetters.java]: Do we really need this variables?

File: src/poonamDPracticeTest/StringNumLetters.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+			if(isNum = Character.isDigit(ch))
```
Comment (mkanani13) [inline src/poonamDPracticeTest/StringNumLetters.java]: Can't we write directly if(Character.isDigit(ch)), why do we need to take it in boolean

File: src/poonamDPracticeTest/StringNumLetters.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+				{
+					Num = Num + ch;
+					int digit = Integer.parseInt(String.valueOf(ch));
```
Comment (mkanani13) [inline src/poonamDPracticeTest/StringNumLetters.java]: Do we need num variable ..

File: src/poonamDPracticeTest/StringNumLetters.java (hunk 2 above)
Comment (PoonamDDec19) [reply src/poonamDPracticeTest/StringNumLetters.java]: I was trying to print all the letters & numbers at first; 
I'll update the program again

**Pull request merged**